File: openapi-generator/csharp-nancyfx/src/Org.OpenAPITools/Models/JobStatus.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;
using Sharpility.Extensions;
using NodaTime;

namespace Org.OpenAPITools.v1.Models
{
    /// <summary>
    /// The current state of the dispatch job.
    /// </summary>
    /// <value>The current state of the dispatch job.</value>
    public enum JobStatus
    {
        
        /// <summary>
        /// Enum Unassigned
        /// </summary>
        Unassigned = 1,
        
        /// <summary>
        /// Enum Scheduled
        /// </summary>
        Scheduled = 2,
        
        /// <summary>
        /// Enum EnRoute
        /// </summary>
        EnRoute = 3,
        
        /// <summary>
        /// Enum Arrived
        /// </summary>
        Arrived = 4,
        
        /// <summary>
        /// Enum Completed
        /// </summary>
        Completed = 5,
        
        /// <summary>
        /// Enum Skipped
        /// </summary>
        Skipped = 6
    }

}

File: swagger-codegen/spring/src/main/java/io/swagger/model/JobStatus.java
package io.swagger.model;

import java.util.Objects;
import io.swagger.annotations.ApiModel;
import com.fasterxml.jackson.annotation.JsonValue;
import org.springframework.validation.annotation.Validated;
import javax.validation.Valid;
import javax.validation.constraints.*;

import com.fasterxml.jackson.annotation.JsonCreator;

/**
 * The current state of the dispatch job.
 */
public enum JobStatus {
  
  UNASSIGNED("JobState_Unassigned"),
  
  SCHEDULED("JobState_Scheduled"),
  
  ENROUTE("JobState_EnRoute"),
  
  ARRIVED("JobState_Arrived"),
  
  COMPLETED("JobState_Completed"),
  
  SKIPPED("JobState_Skipped");

  private String value;

  JobStatus(String value) {
    this.value = value;
  }

  @Override
  @JsonValue
  public String toString() {
    return String.valueOf(value);
  }

  @JsonCreator
  public static JobStatus fromValue(String text) {
    for (JobStatus b : JobStatus.values()) {
      if (String.valueOf(b.value).equals(text)) {
        return b;
      }
    }
    return null;
  }
}

